tests/store/api/test_store_api_status.py

```diff
@@ -229,3 +230,26 @@ def test_multiple_analyses(analysis_store, helpers):
     # THEN only the newest analysis should be returned
     assert analysis_newest in analyses
     assert analysis_oldest not in analyses
+
+
+def test_get_customer_id_from_ticket(analysis_store, ticket_nr):
+    """Tests if the function in fact returns the correct customer."""
+    # Given a store with a ticket
+
+    # Then the function should return the customer connected to the ticket
+    assert analysis_store.get_customer_id_from_ticket(ticket_nr) == "cust000"
+
+
+def test_set_cases_to_analyze(analysis_store, case_id):
+    """Tests if actions of cases are changed to analyze."""
+    # Given a store with a case with action None
+    action = analysis_store.Family.query.filter(Family.internal_id == case_id).first().action
+
+    assert action == None
+
+    # When setting the case to "analyze"
+    analysis_store.set_cases_to_analyze([case_id])
+    new_action = analysis_store.Family.query.filter(Family.internal_id == case_id).first().action
+
+    # Then the action should be set to analyze
+    assert new_action == "analyze"
```

Comment:
Use your newly (hopefully) defined constant (see other comment on "analyze") from the actual code separate from the tests